release notes/v0.37.0.md

```diff
@@ -0,0 +1,73 @@
+k6 v0.37.0 is here! 🎉 Mainly it contains fixes and ongoing efforts with refactoring, but it's just because we are preparing some incredible things in upcoming releases!
+
+## New Features!
+
+### Added support for local files to --log-output ([#2285](https://github.com/grafana/k6/pull/2285))
+
+Added an option to output k6 logs to a file, on top of the supported stdout/stderr and loki instance.
+
+The feature speaks for itself with examples of usage:
+
+```sh
+k6 run --log-output file=./k6.log --logformat json ./examples/stages.js
+```
+
+with a defined minimal log level:
+
+```sh
+k6 run --log-output file=./k6.log,level=info --logformat json ./examples/stages.js
+```
+
+Thanks, @[alyakimenko](https://github.com/alyakimenko) for the contribution!
+
+**Docs**: [Using file output](https://k6.io/docs/using-k6/options/#file)
+
+### Added experimental basic event loops ([#882](https://github.com/grafana/k6/issues/882))
+
+We added basic event loop functionality ([#2228](https://github.com/grafana/k6/pull/2228) and [#2373](https://github.com/grafana/k6/pull/2373)). This is a first step and currently isn't used by anything, but is now available to extensions. In following releases this will be used by k6 and will mean that some APIs will start having callbacks or return Promises. But first it's likely that it will be used in some experimental extensions.
+
+If you are an extension developer please use it and give your feedback. But take into consideration that it's likely that the API may change
+
+## Extensions
+
+`v0.37.0` finalizes ([#2376](https://github.com/grafana/k6/pull/2376) and [#2384](https://github.com/grafana/k6/pull/2384)) the switching of our internal modules to [a new Go/JavaScript module API](https://k6.io/docs/extensions/guides/create-an-extension/#advanced-javascript-extension). As the last internal the GRPC module was refactored ([#2365](https://github.com/grafana/k6/pull/2365)).
```

Comment:
Can we be more specific about which APIs are now deprecated? Extension developers might want to know the specific APIs, without needing to visit the various linked issues.

Reply:
Exposed in https://github.com/grafana/k6/pull/2419/files#diff-70d6df1e5aea876b98647b1c02332746839792866310938aef7d371b79df8be0R35